content/applications/finance/payment_providers/razorpay.rst

```diff
@@ -23,6 +23,14 @@ Configuration on Razorpay Dashboard
    checkboxes are ticked.
 #. Click on :guilabel:`Create Webhook` to finalize the configuration.
 
+.. _payment_providers/razorpay/recurring_payments:
+
+.. important::
+   The :ref:`Recurring payments <payment_providers/razorpay/recurring_payments>` feature must
```

Comment:
The anchor is correct, but the reference to the anchor should be in the view of the provider. Here, this admonition is referencing itself.